### PR TITLE
Add JavaScript ES6 to update folder

### PR DESCRIPTION
**Pull request by flashdesignory**
This version has been lifted from Speedometer and follows the WebKit styles guide.

For more context regarding this update, please view https://github.com/tastejs/todomvc/issues/2199.

@kara

### CODE REVIEW

**Comment by manish-mehra** [comment]
That's exactly what I was looking for. With es6 classes, the code is more readable and I can comprehend it as a newbie.  Thanks @flashdesignory🙇🏻‍♂️. Waiting for the pull request to be merged.

File: updates/javascript-es6/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+[JavaScript - developer.mozilla.org](http://developer.mozilla.org/en-US/docs/JavaScript)
+
+In addition, this application also relies on [Webpack](https://webpack.js.org/) to generate a build. At its core, webpack is a static module bundler for modern JavaScript applications. When webpack processes your application, it internally builds a dependency graph from one or more entry points and then combines every module your project needs into one or more bundles, which are static assets to serve your content from.
+
+## Implementation details
+
+This implementation uses an explicit MVC pattern, with a clear file structure to reflect the architecture. The storage solution uses an in-memory data object that implements a simple array to hold the todos.
+
+## Built steps
```
Comment (FadySamirSadek) [inline updates/javascript-es6/README.md:17]: *build Steps

File: updates/javascript-es6/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+This application uses JavaScript with ES6 language features to implement a todo application.
+
+JavaScript (JS) is a lightweight, interpreted, or just-in-time compiled programming language with first-class functions. While it is most well-known as the scripting language for Web pages, many non-browser environments also use it..
+
+[JavaScript - developer.mozilla.org](http://developer.mozilla.org/en-US/docs/JavaScript)
+
+In addition, this application also relies on [Webpack](https://webpack.js.org/) to generate a build. At its core, webpack is a static module bundler for modern JavaScript applications. When webpack processes your application, it internally builds a dependency graph from one or more entry points and then combines every module your project needs into one or more bundles, which are static assets to serve your content from.
+
+## Implementation details
```
Comment (FadySamirSadek) [inline updates/javascript-es6/README.md:13]: Implementation Details

**Review by FadySamirSadek — CHANGES_REQUESTED** [review]
LGTM, other than the 2 typos in the README.

**Review by FadySamirSadek — APPROVED** [review]

**Pull request merged**